packages/headless/src/features/folding/folding-actions.ts

```diff
@@ -43,3 +61,42 @@ export const registerFolding = createAction(
   (payload: RegisterFoldingActionCreatorPayload) =>
     validatePayload(payload, foldingOptionsSchemaDefinition)
 );
+
+export const loadCollection = createAsyncThunk<
+  LoadCollectionFulfilledReturn,
+  CollectionId,
+  AsyncThunkSearchOptions<
+    ConfigurationSection & FoldingSection,
+    LoadCollectionRejectedReturn
+  >
+>(
+  'folding/loadCollection',
+  async (
+    collectionId: CollectionId,
+    {getState, rejectWithValue, extra: {searchAPIClient}}
+  ) => {
+    const {
+      folding: {fields},
+      configuration: {
+        accessToken,
+        organizationId,
+        search: {apiBaseUrl},
+      },
+    } = getState();
+
+    const response = await searchAPIClient.search({
+      tab: '',
```

Comment:
Should I send an empty tab or the tab from the state?

Reply:
I guess it should be like other parameters, you're not putting the facets since we don't need them to affect the results, should be the same for Tab